src/tests/jungle_logstore/endian_encode.h

```diff
@@ -52,19 +52,18 @@
 
 #ifndef reverse_order_64
 #define reverse_order_64(v)                                                                                            \
-    ((((v) & 0xff00000000000000ULL) >> 56) | (((v) & 0x00ff000000000000ULL) >> 40) |                                   \
-     (((v) & 0x0000ff0000000000ULL) >> 24) | (((v) & 0x000000ff00000000ULL) >> 8) |                                    \
-     (((v) & 0x00000000ff000000ULL) << 8) | (((v) & 0x0000000000ff0000ULL) << 24) |                                    \
-     (((v) & 0x000000000000ff00ULL) << 40) | (((v) & 0x00000000000000ffULL) << 56))
+    ((((v)&0xff00000000000000ULL) >> 56) | (((v)&0x00ff000000000000ULL) >> 40) | (((v)&0x0000ff0000000000ULL) >> 24) | \
+     (((v)&0x000000ff00000000ULL) >> 8) | (((v)&0x00000000ff000000ULL) << 8) | (((v)&0x0000000000ff0000ULL) << 24) |   \
+     (((v)&0x000000000000ff00ULL) << 40) | (((v)&0x00000000000000ffULL) << 56))
 #endif
 
 #ifndef reverse_order_32
 #define reverse_order_32(v)                                                                                            \
-    ((((v) & 0xff000000) >> 24) | (((v) & 0x00ff0000) >> 8) | (((v) & 0x0000ff00) << 8) | (((v) & 0x000000ff) << 24))
+    ((((v)&0xff000000) >> 24) | (((v)&0x00ff0000) >> 8) | (((v)&0x0000ff00) << 8) | (((v)&0x000000ff) << 24))
```

Comment:
There's some funky reformatting going on in this pr. Check the clang tools ? This whitespace removal makes this look awful.

Reply:
I am using the format script in the root dir of the repo. Ill restore this file